.npmrc

```diff
@@ -0,0 +1 @@
+@eccenca:registry=https://npm.pkg.github.com
```

Comment:
is this still required?

Reply:
Correct, should be removed. It won't lead to problems with the package as it is not part of it.